core/src/main/java/com/crazylegend/core/adapters/multi/MultiSelectAdapter.kt

```diff
@@ -15,6 +15,7 @@ import com.crazylegend.core.modifiers.base.BaseMultiPickerModifier
 class MultiSelectAdapter(private val modifier: BaseMultiPickerModifier?) :
         ListAdapter<BaseCursorModel, MultiSelectViewHolder>(SingleDiffUtil()) {
 
+    var showFileSize: Boolean = false
```

Comment:
Pass it through the constructor

Reply:
Got it but can you explain what is wrong with the current approach?

Reply:
No need to do a bi directional connection of the adapter and the fragment.